governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+
+#### *If Emilia means to request a non-standard weekly poll* 
+
+There is no explicit indication that Emilia intended to request a non-standard weekly poll. Assuming she did so intend, the required justification is neither given nor discernible from her post. 
+
+Per MIP16:
+
+> [T]he use of non-standard weekly governance polls will be limited to situations where the [monthly] governance cycle is determined to operate too slowly to be usable. ... To create a non-standard weekly poll requires an urgent and apparent reason from a timing perspective to justify it. 
+
+Per [MIP16c1](https://mips.makerdao.com/mips/details/MIP16#MIP16c1), Emilia is not eligible in any event to make use of this type of poll: the use of non-standard weekly polls is limited to Facilitators. 
+
+#### *If Emilia means to initiate a poll in support of an out-of-schedule MIP24 executive vote*
+
+[MIP24](https://mips.makerdao.com/mips/details/MIP24) defines Maker's emergency voting system, pursuant to which any community member may initiate an emergency executive vote. MIP24 is inapplicable in this situation. Emilia's post does not present an "emergency situation" or an "urgent situation", as defined in [MIP24c1](https://mips.makerdao.com/mips/details/MIP24#MIP24c1).
+
+Granted, Emilia did not explicitly request an out-of-schedule Executive Vote under MIP24. But accounting for the ambiguity in the Exercise prompt, it is possible she was incorrectly trying to invoke one. After all, Emilia was clearly seeking an accelerated timeline. Further, Emilia's proposal combined: 1) the purported invalidation of the current ALM Standing Order (high stakes!), and 2) a Forum poll which may have been an attempted Signal Request. Note that [MIP24c3](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) provides the option of creating a signal request thread or governance poll to initiate the emergency response process.
+
+(Generally, Signal Requests are [no longer permitted](https://mips.makerdao.com/mips/details/MIP4c2SP27#amended-mips-and-components), unless specifically triggered and required by an existing MIP, such as MIP24.)
+
+### Problems with the Standing Orders (aka Instruction Sets)
+
+I assume the Exercises' Standing Orders are equivalent to the "Instruction Sets" of current RWA structures. (See [1](https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969#instruction-sets-9), [2](https://forum.makerdao.com/t/clydesdale-vault-hq/17923#clydesdale-instruction-sets-4).) For the purpose of the Exercise, I use [Monetalis Clydesdale](http://forum.makerdao.com/t/mip65-clydesdale-governance-framework-setup/16565)'s governance framework as a reference point in terms of best practices/prerequisites concerning a Standing Order/Instruction Set:
```

Comment:
Correct, though to be clear, I have no idea how they actually work in practice, so there may have been some obvious weirdness to someone with that familiarity.